docs/content/en/docs/tasks/cluster/manual-renew-certs.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Manual Renew Certificates"
+linkTitle: "Manual Renew Certificates"
+weight: 31
+date: 2021-11-04
+description: >
+  How to renew certificates in etcd and control plane machines
+---
+
+Certificates in external etcd and control plane nodes expire after 1 year. The following table shows all the certificate files:
+
+
+| etcd node             | control plane node       |
+|-----------------------|--------------------------|
+| apiserver-etcd-client | apiserver-etcd-client    |
+| ca                    | ca                       |
+| etcdctl-etcd-client   | front-proxy-ca           |
+| peer                  | sa                       |
+| server                | etcd/ca.crt              |
+|                       | apiserver-kubelet-client |
+|                       | apiserver                |
+|                       | front-proxy-client       |
+
+You can renew all certificates in the table except ca by following the steps given below.
+
+### External etcd node
+
+If the cluster has external etcd, you need to renew the certificates in etcd nodes first.
+
+1. ssh into the etcd node and run the following commands:
+
+{{< tabpane >}}
+{{< tab header="Ubuntu or RHEL" lang="bash" >}}
+# backup certs
+cd /etc/etcd
+sudo mv pki pki.bak
+sudo mkdir pki
+sudo mv pki.bak/ca.* pki
+
+# run certificates join phase to regenerate the deleted certificates
+sudo etcdadm join phase certificates http://dumburl
```

Comment:
Can we explain what this `dumburl` is? Users might feel they should give in a url. Maybe we can say `http://eks-a-etcd-dumb-url`

Reply:
etcdadm join phase is a command that we added through patch, and since this command is a hack, it requires a dumb url otherwise the preflight check would fail